bundles/org.eclipse.ui.ide/src/org/eclipse/ui/internal/views/markers/FiltersConfigurationDialog.java

```diff
@@ -678,6 +691,13 @@ protected void okPressed() {
 	}
 
 	@Override
+	protected void buttonPressed(int buttonId) {
+		if (buttonId == DEFAULTS_BUTTON_ID) {
+			performDefaults();
+		}
+		super.buttonPressed(buttonId);
+	}
+
 	protected void performDefaults() {
```

Comment:
`@Override` should remain here.

Reply:
Its not @Override now that its using TrayDialog.

Reply:
Ok, my bad.